src/components/Page/Page.test.tsx

```diff
@@ -37,4 +38,18 @@ describe('Page', () => {
       });
     });
   });
+
+  Object.values(PageSize).forEach((size) => {
+    it(`should render a PageHeader with ${size} classname when size is ${size}`, () => {
+      render({ size });
+      const otherSizes = Object.values(PageSize).filter((s) => s !== size);
+
+      const pageHeader = screen.getByTestId('page-header');
```

Comment:
Unngå å bruke `getByTestId` hvis du ikke må.
```suggestion
      const pageHeader = screen.getByRole('heading');
```

Reply:
So I should convert component PageHeader to a <div> instead of having a <header> and add role='heading' and aria-level='1' to that div?

Reply:
I meant `banner`, not `heading`, my bad. The `<header>` tag has this role by default, so you shouldn't have to change anything in the actual component. Using `getByRole` makes the test more aligned with the actual user experience, hence why we should stick to that when possible. The user doesn't care about the test id.

Reply:
I see, I agree that we should use getByRole when we can :)